### PR TITLE
elide usernames in timeline

### PR DESCRIPTION
**Pull request by maltee1**
Elide usernames above messages and in replies. Please review & merge.

### CODE REVIEW

**Comment by deepbluev7** [comment]
This looks like it elides too short:

grafik

But I guess that is for the status message?

grafik

But if I then narrow the timeline:

grafik

It overflows?

File: resources/qml/ElidedLabel.qml [hunk 1]
```diff
@@ -12,6 +12,7 @@ Label {
 
     property alias fullText: metrics.text
     property alias elideWidth: metrics.elideWidth
+    property int fullTextWidth: metrics.width+1 // prevent premature eliding
```
Comment (deepbluev7) [inline resources/qml/ElidedLabel.qml:15]: Maybe Math.ceil works too?

**Comment by maltee1** [comment]
Math.ceil is probably a good idea.
The problems with the width are probably due to other parts of the layout. Since I planned a major rework of that, maybe it makes sense to just push these changes and see if the width problems get fixed in the process.

**Comment by deepbluev7** [comment]
Probably makes sense to just use this branch. It fixes 2 things and introduces another weirdness :D

**Comment by deepbluev7** [comment]
It still had the premature elision issues for me, but using the advanceWidth seems to fix that :3

**Pull request merged**